2019-01-02 16:40:33.829 dirwatcher.py INFO    
-------------------------------------------------------------------
    Running dirwatcher.py
    Started on 2019-01-02T16:40:33.829397
-------------------------------------------------------------------

2019-01-02 16:40:33.829 dirwatcher.py INFO    Running this directory watcher function
2019-01-02 16:40:42.840 dirwatcher.py WARNING Received signal number 2
2019-01-02 16:40:42.840 dirwatcher.py WARNING done with this program
2019-01-02 16:40:42.841 dirwatcher.py INFO    
-------------------------------------------------------------------
   Stopped dirwatcher.py
   Uptime was 0:00:09.011662
-------------------------------------------------------------------

2019-01-02 16:40:52.756 dirwatcher.py INFO    
-------------------------------------------------------------------
    Running dirwatcher.py
    Started on 2019-01-02T16:40:52.756404
-------------------------------------------------------------------

2019-01-02 16:40:52.756 dirwatcher.py INFO    Running this directory watcher function
2019-01-02 16:40:53.758 dirwatcher.py INFO    File: a.txt At line: 2 Text: but really not
2019-01-02 16:40:53.758 dirwatcher.py INFO    File: a.html At line: 1 Text: really
2019-01-02 16:41:10.299 dirwatcher.py WARNING Received signal number 2
2019-01-02 16:41:10.300 dirwatcher.py WARNING done with this program
2019-01-02 16:41:10.300 dirwatcher.py INFO    
-------------------------------------------------------------------
   Stopped dirwatcher.py
   Uptime was 0:00:17.543956
-------------------------------------------------------------------

2019-01-02 16:42:37.337 dirwatcher.py INFO    
-------------------------------------------------------------------
    Running dirwatcher.py
    Started on 2019-01-02T16:42:37.337214
-------------------------------------------------------------------

2019-01-02 16:42:37.337 dirwatcher.py INFO    Running this directory watcher function
2019-01-02 16:42:38.338 dirwatcher.py ERROR   Unhandled exception: local variable 'file_lines' referenced before assignment
2019-01-02 16:42:38.339 dirwatcher.py INFO    Retrying in 5 seconds ...
2019-01-02 16:42:43.167 dirwatcher.py WARNING Received signal number 2
2019-01-02 16:42:43.168 dirwatcher.py WARNING done with this program
2019-01-02 16:42:43.168 dirwatcher.py INFO    
-------------------------------------------------------------------
   Stopped dirwatcher.py
   Uptime was 0:00:05.831453
-------------------------------------------------------------------

2019-01-02 16:43:23.130 dirwatcher.py INFO    
-------------------------------------------------------------------
    Running dirwatcher.py
    Started on 2019-01-02T16:43:23.130963
-------------------------------------------------------------------

2019-01-02 16:43:23.131 dirwatcher.py INFO    Running this directory watcher function
2019-01-02 16:43:24.132 dirwatcher.py INFO    File: a.txt At line: 2 Text: but really not
2019-01-02 16:43:24.133 dirwatcher.py INFO    File: a.html At line: 1 Text: really
2019-01-02 16:45:16.448 dirwatcher.py WARNING Received signal number 2
2019-01-02 16:45:16.448 dirwatcher.py WARNING done with this program
2019-01-02 16:45:16.449 dirwatcher.py INFO    
-------------------------------------------------------------------
   Stopped dirwatcher.py
   Uptime was 0:01:53.318174
-------------------------------------------------------------------

